contracts/payloads/IGP27/PayloadIGP27.sol

```diff
@@ -0,0 +1,453 @@
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Update rewards for fUSDC & fUSDT.
+        action1();
+
+        // Action 2: Update USDC, USDT & wstETH market rate curve.
+        action2();
+
+        // Action 3: Set max utilization to 85% for wstETH & Update Reserve fee to 10%.
+        action3();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Update rewards for fUSDC & fUSDT.
+    function action1() internal {
+        IFTokenAdmin(F_USDC).updateRewards(
+            0xf6109Ad48f5Cf9e28938774aD4069C90f81D648E
+        );
+
+        IFTokenAdmin(F_USDT).updateRewards(
+            0xf6109Ad48f5Cf9e28938774aD4069C90f81D648E
+        );
+    }
+
+    /// @notice Action 2: Update USDC, USDT & wstETH market rate curve.
+    function action2() internal {
+        AdminModuleStructs.RateDataV2Params[]
+            memory params_ = new AdminModuleStructs.RateDataV2Params[](3);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
+            token: USDC_ADDRESS, // USDC
+            kink1: 80 * 1e2, // 80%
+            kink2: 93 * 1e2, // 93%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 12 * 1e2, // 12%
+            rateAtUtilizationKink2: 18 * 1e2, // 18%
+            rateAtUtilizationMax: 33.33 * 1e2 // 33.33%
+        });
+
+        params_[1] = AdminModuleStructs.RateDataV2Params({
+            token: USDT_ADDRESS, // USDT
+            kink1: 80 * 1e2, // 80%
+            kink2: 93 * 1e2, // 93%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 12 * 1e2, // 12%
+            rateAtUtilizationKink2: 18 * 1e2, // 18%
+            rateAtUtilizationMax: 33.33 * 1e2 // 33.33%
+        });
+
+        params_[2] = AdminModuleStructs.RateDataV2Params({
+            token: wstETH_ADDRESS, // wstETH
+            kink1: 50 * 1e2, // 50%
+            kink2: 90 * 1e2, // 90%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 15 * 1e2, // 15%
+            rateAtUtilizationKink2: 12 * 1e2, // 12%
+            rateAtUtilizationMax: 100 * 1e2 // 100%
+        });
+
+        LIQUIDITY.updateRateDataV2s(params_);
+    }
+
+
+    /// @notice Action 3: Set max utilization to 85% for wstETH & Update Reserve fee to 10%.
+    function action3() internal {
+        AdminModuleStructs.TokenConfig[]
+            memory tokenConfigs_ = new AdminModuleStructs.TokenConfig[](2);
+
+        tokenConfigs_[0] = AdminModuleStructs.TokenConfig({
+            token: wstETH_ADDRESS,
```

Comment:
This should be for native ETH from what I understand of https://gov.instadapp.io/t/update-usdc-usdt-and-wsteth-market-rates-on-fluid-jun-24-2024/780

Also, for wstETH it is already 10% but for ETH currently it is 0% so that would make more sense. 

So for ETH it should be:

```
token: ETH_ADDRESS,
fee: 10 * 1e2, // 10%
threshold: 0.3 * 1e2, // 0.3%
maxUtilization: 100 * 1e2 // 100%
```